### DISCUSSION TITLE
Your request was not processed as expected. Please resend your request. If you experience the same issue, you will need to contact us by other means.

### DISCUSSION THREAD

**Discussion by Broninn** [LiveHelperChat/livehelperchat#2030] [Q&A] [answered]
Hello everyone,

I put the bot online, and every now and then the following message appears: xxxx.
"Your request was not processed as expected. Please resend your request. If you experience the same issue, you will need to contact us by other means."
![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/58676456-31f5-4b5a-bee8-13af766ca037)

I have already cleared the cache via the interface and via the server, but it still appears for some users

Has anyone ever experienced this?

**Comment by remdex** [comment, marked as answer]
Most likely you messed around with settings and enabled session based captcha in settings... Disable it.

**Comment by Broninn** [comment]
Hello, I checked the day about recaptcha, and it was disabled. I cleared the cache and it had worked, but today it appeared again, and the recaptcha is disabled.

The configuration I did was just build a bot, create triggers, and at the end of each flow you have the possibility to talk to the attendant, but without using recaptcha

![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/776a5126-a57c-4cc8-b478-e267ae3bfb6c)

**Reply by remdex** [reply]
That has nothing to do with that section. It's for back office login enhancements. For you those are related. Make them as bellow.
![image](https://github.com/LiveHelperChat/livehelperchat/assets/1146085/0563500b-cd9b-4134-bc86-92080543fa8d)

**Comment by Broninn** [comment]
Hello, I checked the indicated settings and they were already like this too

![image](https://github.com/LiveHelperChat/livehelperchat/assets/65509910/9bf63924-a02b-4450-8ae1-8966cdab2ac3)

**Comment by remdex** [comment]
In that case it's your env problem. For personall support there is paid service...

**Comment by Broninn** [comment]
Hello. To resolve this issue, just needed set this variable true. We don't use captcha, so it's works.

lib/core/lhchat/lhchatvalidator.php" row 293 
"$additionalParams['ignore_captcha'] = true;

![image](https://github.com/user-attachments/assets/21744ab6-c179-438f-beed-74e17609adf2)

**Reply by abdullasawood** [reply]
Thanks a lot. 
Lack of solution for the said issue online was stumping for sure.
However as remdex mentioned, it may not be optimal solution, but it is the only solution that works at the moment.

The re-captcha is disabled and use session captcha under chat configuration -> misc is also disabled. Yet If LHC is and widgets are on different servers, I was getting the error:
Your request was not processed as expected - but don't worry it was not your fault. Please re-submit your request. If you experience the same issue you will need to contact us via other means.

In network tab submitonline request showed response - 
"captcha" : "Your request was not processed as expected - but don't worry it was not your fault. Please re-submit your request. If you experience the same issue you will need to contact us via other means."

I do have my website cached and firewall by cloudflare.

As per chatgpt:
lhc_vid and PHPSESSID must not change between:
widgetrestapi/settings → submitonline

Cloudflare caching must not cache PHP endpoints:
/lhc_web/index.php/widgetrestapi/*

But they look a bit complex to look into. so this solution is a quick fix to a solution that I don't want to integrate (captcha).

Sorry for the long comment. I thought I'd just put it out there for anyone are facing similar issue or if the devs may want to check/fix this in future.

**Comment by remdex** [comment]
That's incorrect solution which reduces security of LHC...
